xbmc/cores/VideoPlayer/VideoRenderers/OverlayRendererUtil.cpp

```diff
@@ -159,21 +159,18 @@ std::vector<uint32_t> convert_rgba(
     max_y = max_x = 1;
     min_y = min_x = 0;
   }
-
-  return rgba;
 }
 
 bool convert_quad(ASS_Image* images, SQuads& quads, int max_x)
 {
   ASS_Image* img;
+  int count = 0;
 
   if (!images)
     return false;
 
   // first calculate how many glyph we have and the total x length
 
-  int count{0};
```

Comment:
Did you mean to move this? It seems irrelevant?

Reply:
All changes from PR20260 should not be interpreted as fixups or things that were wrong. I have started from the functional code in Matrix without looking much at the code of the previous PR (so as not to be influenced and make the same mistakes). So the resulting code has been different, which on the other hand is what is expected and what I intended.

Once this has been effectively justified, the change is irrelevant and for this same reason it can also be left as it is.